src/geovista/examples/warp/README.rst

```diff
@@ -1,2 +1,25 @@
 Warp Mesh
 =========
+
+These examples show how to make plots where an elevation offset is applied to
+each mesh point -- i.e. the mesh has a visible "height" above the surface.
+
+In ``pyvista``, applying an offset to each point is called a "warp".
+In particular, the function :meth:`~pyvista.DataSetFilters.warp_by_scalar` is
+used to displace each point by some amount in the direction of the mesh
+surface normals.
+
+.. note::
+    The mesh normals must first be computed,
+    using :meth:`~pyvista.PolyDataFilters.compute_normals`.
+
+See `Computing Surface Normals <https://docs.pyvista.org/version/stable/examples/01-filter/compute-normals.htm>`_
+for a fuller description of warping in general.
+
+For a *cartographic* mesh, naturally, the surface normals give the upward
+direction, and the warp is simply a vertical displacement.
+
+The scalar warping array is often elevation data, usually with a scale factor
+for emphasis.
+:ref:`geotiff-elevation-example` is a good demonstration of this.
+However, you can of course display any scalar mesh data as a relief in this way.
```

Comment:
@pp-mo This is fantastic detail 😄 However it unfortunately interrupts the flow of the gallery

<details>
<summary>See here ...</summary>

![image](https://github.com/bjlittle/geovista/assets/2051656/82e0cf25-4d1b-4b8a-8701-68e7cc63a699)

</details>

I really wanted to keep the main gallery page clean, and draw the users to the examples simply through eye candy images ... otherwise the gallery landing page will get way too busy.

TBH this is the kind of lovely detail that I'd like to see either as part of a tutorial on warping or in the how-to section.

As we've not shaped the structure of either of those sections yet, how about we capture your contribution in an issue (so it's not lost) as example detail we want to include in the docs in the near future.

Does that sound reasonable?

Reply:
Ok but the main reason I bother with this was because when I looked at the individual examples they were all taking for granted what "warp" actually meant.  So I though that was a geovista thing, and then eventually worked out that it was a pyvista term.

So perhaps we could inject a severely cut-down section into all 3/4 examples to explain what the concept is.  A link to the pv "Computing Surface Normals" section would be the key thing.

How does that sound ?

Reply:
Yeah, apologies for the faff 😞 

Ideally the examples would all link to deeper explanations elsewhere in the docs ... but since we don't have that just yet, you're suggested compromise of adding some replicated summary detail to each of them sounds totally pragmatic to me.

Go for it 👍

Reply:
> Yeah, apologies for the faff 😞

Equally. sorry for all the stream-of-consciousness questions !

I think I have got your comments now, this + others.
I will work it out from there + come up with new proposals.